server/pr_from_issue_handler.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package server
+
+import (
+	"database/sql"
+	"net/http"
+
+	"github.com/mattermost/mattermost-server/v5/mlog"
+)
+
+func (s *Server) prFromIssueHandler(event *issueEvent, w http.ResponseWriter) {
+	oldPR, err := s.Store.PullRequest().Get(event.Repo.GetOwner().GetLogin(),
+		event.Repo.GetName(),
+		event.Issue.GetNumber())
+	if err != nil {
+		mlog.Error("Error in getting PR from DB", mlog.Err(err))
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
```

Comment:
Non-blocker: 
I think we should get the PR from github if we don't already have it, instead of returning error directly.

Reply:
This handler will only be called when adding a milestone for an existing PR. A new PR would already be created in the DB from other handlers. So I think it should be fine. I will obviously monitor the logs if this happens and will add the case if needed.